x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/configure_custom.tsx

```diff
@@ -57,20 +59,20 @@ export const ConfigureCustom: React.FC<ConfigureCustomProps> = ({
             <p>{helpText}</p>
             <p>
               <FormattedMessage
-                id="xpack.enterpriseSearch.workplaceSearch.contentSource.configCustom.docs.link"
+                id="xpack.enterpriseSearch.workplaceSearch.contentSource.configCustom.docs.link.description"
```

Comment:
As per the i18n guidelines, can we change this to:
```suggestion
                id="xpack.enterpriseSearch.workplaceSearch.contentSource.configCustom.docs.linkDescription"
```

Reply:
RE: "id format" replied here: https://github.com/elastic/kibana/pull/112136#issuecomment-919985697